Modules/pyexpat.c

```diff
@@ -1848,7 +1791,124 @@ MODULE_INITFUNC(void)
     MYCONST(XML_CQUANT_REP);
     MYCONST(XML_CQUANT_PLUS);
 #undef MYCONST
+    return 0;
+}
+
+static int add_features(PyObject *mod) {
```

Comment:
```suggestion
static int
add_features(PyObject *mod)
{
```

If suggest to put the whole function between #if...#endif, and also add these #if..#endif around the add_features(mod) call.